AEPCore/Sources/eventhub/EventListenerContainer.swift

```diff
@@ -44,8 +44,8 @@ struct EventListenerContainer: Equatable {
     /// - Parameter event: An `Event` being dispatched by the `EventHub`
     /// - Returns: True if `listener` should be notified of `event`
     func shouldNotify(_ event: Event) -> Bool {
-        if let listenerTriggerId = triggerEventId {
-            return listenerTriggerId == event.responseID
+        if event.responseID != nil {
+            return event.responseID == triggerEventId || self.isWildcard()
         }
 
         return (event.type == type || type == EventType.wildcard)
```

Comment:
Can we update the last return statement to use the `isWildcard` member?
`return (event.type == type && event.source == source) || isWildcard`

Reply:
not really, because the current condition also covers the use-case where only one (type or source) is wildcard.